py3status/modules/google_calendar.py

```diff
@@ -51,8 +56,17 @@
         (default False)
     num_events: The maximum number of events to display.
         (default 3)
+    preferred_event_link: link to open in the browser.
+        accepted values :
```

Comment:
I'm sorry if my previous comment was not clear enough but here is where I expected to be less strict and point to any accepted value found (or which could be added later) in the google API response (just like you did with the response config below

So instead of listing accepted values, list them as examples but state clearly that any value present in the xxxx.xxxx google api object is valid there and add a link to its docs?

Reply:
Unfortunately, we cannot tell "any value present in this SPECIFIC-FIELD", because we can retrieve a value from various fields : 
- htmlLink is located at .htmlLink of the Google API Object
- hangoutLink is located at .hangoutsLink of the Google API Object.

There is no obvious rule that can be given for (any value of the following object) -  this is why I considered safer for new users to explicitly tell what are the possible values. 
Maybe tomorrow Google will introduce a .newServiceURL in their scheme, at another location.

I'm sorry if I'm not clear enough --  I don't see how to document better, I have included  a link to Google Calendar API's to the top of the documentation.

Please feel free to reach me in DM if required.

Reply:
thanks!